specification/customer-insights/resource-manager/Microsoft.CustomerInsights/2017-01-01/customer-insights.json

```diff
@@ -330,29 +404,26 @@
         ],
         "responses": {
           "200": {
-            "description": "OK. Successfully get all the hubs in the subscription.",
+            "description": "OK. Successfully get the connector operation.",
             "schema": {
-              "$ref": "#/definitions/HubListResult"
+              "$ref": "#/definitions/ConnectorResourceFormat"
```

Comment:
Why is GET on operation resource returning a connector resource. What is the intent of this API? What operation is this? RBAC issue.